src/events/cd-event-list.vue

```diff
@@ -88,7 +131,10 @@
     },
     created() {
       if (this.dojo.verified) {
-        this.loadEvents();
+        this.loadEvents()
```

Comment:
`this.futureEvents = (await this.loadEvents()).body.results;`?

Reply:
No. We don't want the calls to be blocking